manager/controllers/advisory_detail.go

```diff
@@ -0,0 +1,55 @@
+package controllers
+
+import (
+	"github.com/gin-gonic/gin"
+	"net/http"
+	"time"
+)
+
+type AdvisoryDetailResponse struct {
+	Attributes AdvisoryDetailAttributes `json:"attributes"`
+	ID         string                   `json:"id"`
+	Type       string                   `json:"type"`
+}
+
+type AdvisoryDetailAttributes struct {
+	Description  string     `json:"description"`
+	Severity     *string    `json:"severity"`
+	ModifiedDate time.Time  `json:"modified_date"`
+	PublicDate   time.Time  `json:"public_date"`
+	Topic        string     `json:"topic"`
+	Synopsis     string     `json:"synopsis"`
+	Solution     string     `json:"solution"`
+	Fixes        *string    `json:"fixes"`
+	Cves         []string   `json:"cves"`
+	References   []string   `json:"references"`
+}
+
+// @Summary Show me details an advisory by given advisory name
+// @Description Show me details an advisory by given advisory name
+// @Accept   json
+// @Produce  json
+// @Param    advisory_id    path    string   true "Advisory ID"
+// @Success 200 {object} AdvisoryDetailResponse
+// @Router /api/patch/v1/advisories/{advisory_id} [get]
+func AdvisoryDetailHandler(c *gin.Context) {
```

Comment:
This method should return a struct, which contains the `attributes` `id` and `type` in a `data` field, so that the format is the same between `/systems/id` and `/advisories/id`

Reply:
no, because attributes of systems differ to attributes of advisory

Reply:
I don't mean the attributes, I mean the document structure. Both of them should be

```json
{
    "data": {
        "attributes" : {
        
        },
        "id" : "<id>",
        "type" : "<type>"
    }
}
```

Reply:
Ok, I see it now, thanks and sorry.